.github/ISSUE_TEMPLATE.md

```diff
@@ -1,25 +1,30 @@
 <!--Please use the template provided below when reporting bugs:-->
-## Link to bug demonstration repository.
+## Link to bug demonstration repository
 <!--
 A demo repository will show the combination of module versions being used,
 command-lines and configurations.  NYC is very complex, it is difficult or
 even impossible to triage most bugs without this information.
 
 If you cannot share a link to your repository (maybe it's private) then you must
-create a basic `nyc-bug-demo` repository and link to that please.
+create a basic `nyc-bug-demo` repository and link to that please [mandatory].
 -->
 
 ## Expected Behavior
 
 ## Observed Behavior
 
-## Forensic Information
+### Troubleshooting steps
+- [ ] still occurrring when I put `cache: false` in my nyc config
 
-**Operating System:** the operating system you observed the issue on.
-**Environment Information:** information about your project's environment, see instructions below:
-
-1. run the following script:
+## Environment Information
+<!--
+[mandatory] run the following script: npx envinfo --preset nyc
 
-sh -c 'node --version; npm --version; npm ls' > output.txt
+OR:
```

Comment:
I'd honestly just point out the `npx` command. this is really slick.

Reply:
Just wanted to accommodate those who might not have npx yet... do you feel strongly about this though? If you do I will make the change.

Reply:
npx is included in node.js 8+.  nyc@14 will support node.js 6 so I lean towards leaving the global install method listed for now.  I think we should remove the instructions at some point in the future, maybe once we have a second semver-major that does not support node.js 6?

Side note: I actually have `alias npx='npx --no-install'` in my bashrc, so I have to use `npm i -g envinfo` even through I'm running node.js 10.  That said I know that I explicitly disabled the auto-install function of npx so I also know how to deal with the consequences of that.

One potential concern for the global install, will `yarn global add envinfo` update an existing envinfo package?  I know `npm i -g envinfo` does update.

Reply:
I removed it, just to make it more consistent, using `npx envinfo@latest` as @bcoe and @tabrindle suggested :)